client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,24 @@
-import React from 'react';
+import React from "react";
 
 const MovieCard = props => {
-  return;
+  return (
```

Comment:
another approach for the refactor is to move the destructuring inside the MovieCard.